app/controllers/posts_controller.rb

```diff
@@ -72,6 +66,16 @@ def destroy
 
   private
 
+  def assign_post_variables
+    if admin?
+      @posts = Post.order('published_at DESC').page(params[:page]).per_page(19)
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.